image-processing/main.py

```diff
@@ -63,3 +63,27 @@ def download_image(bucket_name, source_blob_name, destination_file_name):
     # using `Bucket.blob` is preferred here.
     blob = bucket.blob(source_blob_name)
     blob.download_to_filename(destination_file_name)
+
+def upload_processeed_images(bucket_name, source_file_name, destination_blob_name):
+    """Uploads images to the bucket.
+    Args:
+        bucket_name = "your-bucket-name"
+        source_file_name = A list of file names to upload
+        destination_blob_name = A list of blob names in corresponding to file names
```

Comment:
likewise, I don't think we need to store a list of destination names which is kinda inefficient. Each source image should translate to a destination image and this translation should be documented so firebase team knows how to fetch them for their survey.